### ISSUE TITLE
Cystic acne

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6219]
**Preferred term label:**
Cystic acne

**Synonyms**

**Definition (free text, please give PubMed ID)**
A severe type of acne characterized by the formation of cysts enclosing a mixture of keratin and sebum.

**Parent term (use hpo.jax.org/app)**
Acne

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 604416 ,  PMID: 9212761 n=7/10, juvenile onset

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP_0033188
